typescript/infra/src/core/deploy.ts

```diff
@@ -28,8 +25,8 @@ export class AbacusCoreInfraDeployer<
         inboxes: {},
         outbox: {
           addresses: {
-            outbox: contracts.outbox.address,
-            interchainGasPaymaster: contracts.interchainGasPaymaster.address,
+            outbox: contracts?.outbox?.address,
+            interchainGasPaymaster: contracts?.interchainGasPaymaster?.address,
```

Comment:
can we just make `deployedContracts` a `Partial<Contracts>`

Reply:
it is no?